src/EMG.Lambda.LocalRunner/IRunnerBuilder.cs

```diff
@@ -13,6 +13,8 @@ public interface IRunnerBuilder
         IRunnerBuilder UseSerializer<TSerializer>(Func<TSerializer> serializerFactory)
             where TSerializer : ILambdaSerializer;
 
+        IRunnerBuilder WithResponseContentType(string contentType);
```

Comment:
Shouldn't this one belong to IReturningRunnerBuilder? Ideally I would first set the return type and then specify the content type, or?

Reply:
Yeah, it makes sense that the content type only makes sense for a lambda that returns something